### PR TITLE
Assignment4 - c200071

### PR DESCRIPTION
**Pull request by pedresp**

### CODE REVIEW

**Comment by anaigmo** [comment]
You are missing the rdflib alternatives in tasks 7.1-3. Apart from that, the rest is looking good

**Comment by pedresp** [comment]
Added the missing rdflib alternatives to the task07.py file.

File: Assignment4/pedresp-c200071/task07.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+"""**Task 7.5: List the entities who know at least two other entities in the graph**"""
+
+# TO DO
+q5 = prepareQuery('''
+  select ?ent where{
+    ?ent foaf:knows ?ent2.
+  }
+  group by ?ent
+  having (count(?ent)>1)
```
Comment (anaigmo) [inline Assignment4/pedresp-c200071/task07.py:145]: you are extracting two entities (?ent) that know the same entity (?ent2), but it's the other way around

**Comment by anaigmo** [comment]
Entregado con errores sin corregir, no se aceptan más PR

**Pull request merged**